src/Core/Checkout/Promotion/Subscriber/PromotionIndividualCodeRedeemer.php

```diff
@@ -83,7 +105,7 @@ public function onOrderPlaced(CheckoutOrderPlacedEvent $event): void
                         'payload' => $individualCode->getPayload(),
```

Comment:
same here then, we do update calls in a loop, I would prefer collection and one DB call